examples/example_mountain_car.py

```diff
@@ -55,30 +51,30 @@ def __init__(self, idx, inputs):
 # episodes.
 
 
-def inner_objective(f, seed, n_total_steps, *, render):
-
-    np.random.seed(seed)
+def inner_objective(f, seed, n_trials_per_individual, n_total_steps, *, render):
```

Comment:
Wouldn't `n_episodes_per_individual` be proper RL-lingo?

Reply:
ah I saw below that fast agents have more episodes, if they don't use all time steps. But I think it is a bit confusing that trial is bigger than episode. Maybe `n_runs_per_individual`

Reply:
yeah, this might indeed be a bit confusing naming. i'll use your suggestion :+1: